package.json

```diff
@@ -4,12 +4,11 @@
   "version": "3.11.0",
   "author": "Salesforce",
   "bin": {
-    "oclif": "bin/run",
```

Comment:
these were kinda handy for testing installers.  Any reason they need to be removed?

Reply:
@mshanemc I didn't like the fact that we were exposing an additional bin for no reason other than tests so I modified the tests to be able to work without needing to have oclif2 bin defined here.